.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -0,0 +1,43 @@
+# .NET Core Release Checklist
+
+## Release Versions
+
+_The set of .NET Core versions that are being released as a unit._
+
+* 2.1.X: <runtime/SDK>
+* 3.1.X: <runtime/SDK>
+
+
+
+1. - [ ] Incase of 2.x, run auto-update(refer docs). Incase of 3.X:
+      - [ ] if maestro auto-PR is active, then verify SHA1s in version.Details with manifest versions in VSU share dir (refer docs), make sure that we match the versions
+      - [ ] in-case auto-PR updates are unavailable, checkout a local branch for the pertinent version and run darc updates(refer docs). 
+          - [ ] Push this local branch upstream and start the release PR for the N version that is to be released.
+1. - [ ] Change the `PrivateSourceBuiltArtifactsPackageVersion` in `eng/Versions.props` to match N-1 release artifacts
+1. - [ ] Make sure that `source-build/ProdConFeed.txt` contains the latest feed required for the release. For info on latest feed (refer docs)
+1. - [ ] Verify that we are building with the appropriate stability and release tag (either none for RTM, or "preview1" etc).  This can apply to the SDK itself as well as the core-setup runtime component
+        
+            dotnet --info
+            dotnet --version
+
+      - [ ] Verify ASP.NET package versions used in templates are correct (dependencies.props may need to be updated)
+      - [ ] Verify MS.NETCore.App packages versions used in downlevel templates are correct (dependencies.props may need to be updated)
```

Comment:
How? Is it enough to check the projects that the smoke test generates or should we directly inspect the files in the SDK?

Reply:
Inspecting the files should be a better way to go. @crummel any comments on this?

Reply:
Looking at the project.assets.json is probably what I would say is easiest, whether that's a smoke-test project or a manual one.